packages/ketcher-core/src/application/editor/actions/sgroup.ts

```diff
@@ -91,6 +91,29 @@ export function setExpandSGroup(restruct, sgid, attrs) {
   return action.perform(restruct)
 }
 
+// todo delete after supporting expand - collapse for 2 attachment points
```

Comment:
Please check if this comment should be removed. If not, my personal opinion is to create a new issue for that, because most todos in the comments are seldom done (if you search 'todo' in our codebase, you'll see more than 200 items😔).

Reply:
Added a task for this: https://github.com/epam/ketcher/issues/2603